### PR TITLE
Update cell selection topic

### PR DESCRIPTION
**Pull request by MayaKirova**
Add snippets and styling section.

### CODE REVIEW

File: doc/en/components/grids/_shared/cell-selection.md [hunk 1]
```diff
@@ -88,10 +88,41 @@ If you want to disable cell selection you can just set `CellSelection` to **none
...
+        object[] data = await this.grid.GetSelectedDataAsync(true, true);
+    }
+}
+```
+<!-- end: Blazor -->
+
+<!-- Angular, WebComponents -->
+`getSelectedData` will return array of the selected data in format depending on the selection. Examples below:
 
 1. If three different single cells are selected:
```
Comment (mddragnev) [inline doc/en/components/grids/_shared/cell-selection.md:127]: Those are shown not as list 
Screenshot 2023-05-16 at 14 03 26

File: doc/en/components/grids/_shared/cell-selection.md [hunk 2]
```diff
@@ -88,10 +88,41 @@ If you want to disable cell selection you can just set `CellSelection` to **none
...
+
+`selectRange` - Select a range of cells with the API. rowStart and rowEnd should use row indexes and columnStart and columnEnd could use column index or column data field value.
+
+```ts
+const range = { rowStart: 2, rowEnd: 2, columnStart: 1, columnEnd: 1 };
+this.grid.selectRange(range);
+```
+
+<!-- end: Angular, WebComponents -->
+
```
Comment (mddragnev) [inline doc/en/components/grids/_shared/cell-selection.md:104]: Clear cell selection api method information is missing for both WC and Blazor: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/cell-selection#clear-cell-selection

File: doc/en/components/grids/_shared/cell-selection.md [hunk 3]
```diff
@@ -88,10 +88,41 @@ If you want to disable cell selection you can just set `CellSelection` to **none
 
 Below are the methods that you can use in order to select ranges, clear selection or get selected cells data.
 
-<!-- Angular -->
+
+<!-- Angular, WebComponents -->
+
+### Select range
+
+`selectRange` - Select a range of cells with the API. rowStart and rowEnd should use row indexes and columnStart and columnEnd could use column index or column data field value.
```
Comment (mddragnev) [inline doc/en/components/grids/_shared/cell-selection.md:96]: This is missing for Blazor, is it not working?

**Review by mddragnev — CHANGES_REQUESTED** [review]

File: doc/en/components/grids/_shared/cell-selection.md (hunk 2 above)
Comment (MayaKirova) [reply doc/en/components/grids/_shared/cell-selection.md:104]: It does not seem to be exposed in angular elements grid. Throws error:
clearCellSelection is not a function

File: doc/en/components/grids/_shared/cell-selection.md (hunk 3 above)
Comment (MayaKirova) [reply doc/en/components/grids/_shared/cell-selection.md:96]: Yes -> https://infragistics.visualstudio.com/NetAdvantage/_workitems/edit/23387/

File: doc/en/components/grids/_shared/cell-selection.md (hunk 1 above)
Comment (MayaKirova) [reply doc/en/components/grids/_shared/cell-selection.md:127]: Formatting should be fixed now.

File: doc/en/components/grids/_shared/cell-selection.md (hunk 2 above)
Comment (mddragnev) [reply doc/en/components/grids/_shared/cell-selection.md:104]: Do we have that logged?

**Review by mddragnev — APPROVED** [review]

**Comment by dkamburov** [comment]
The build is failing for that one, seems to be a broken link

**Review by dkamburov — APPROVED** [review]

**Pull request merged**